ci-jobs/functional/run_appium.yml

```diff
@@ -0,0 +1,25 @@
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: Install Node 10.x
+- script: npm install -g appium
+  displayName: Install appium
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.x'
+- script: appium --relaxed-security &
+  displayName: Run Appium in background
+- script: brew install ffmpeg
+  displayName: Resolve dependencies (Appium server)
+- script: python setup.py install
```

Comment:
What about calling only `pipenv` like install CI related dependencies by `ci-requirements.txt`?

Reply:
Used existing `Pipfile`, it's too enough for ci, though.
`Pipfile` seems impossible to have another section for CI like package/dev-package.